ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/snmp-settings.j2

```diff
@@ -52,12 +52,12 @@ snmp-server host {{ host['host'] }} {% if host['vrf']is defined %}vrf {{ host['v
 {%              endif %}
 {%          endfor %}
 {%    endif %}
-{%    if snmp_server.traps is defined and snmp_server.traps is not none %}
-{%       if snmp_server.traps.enable is defined and snmp_server.traps.enable == true %}
+{%    if snmp_server.traps is arista.avd.defined %}
```

Comment:
this check can be removed since arista.avd.defined will verify the full variable path